frontend/src/component/project/Project/ProjectEnterpriseSettingsForm/ProjectEnterpriseSettingsForm.tsx

```diff
@@ -0,0 +1,367 @@
+import React, { useEffect } from 'react';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import Select from 'component/common/select';
+import { ProjectMode } from '../hooks/useProjectEnterpriseSettingsForm';
```

Comment:
Can be shortened

Reply:
This is actually a hook in the parent folder

Reply:
but can be aliased if the dots are the problem